packages/satoshi-kit/src/widgets/account-button/AccountButton.tsx

```diff
@@ -29,23 +29,25 @@ export const AccountButton = ({
 	children,
 }: AccountButtonProps) => {
 	const { config } = useSatoshiKit();
-	const { accounts } = useAccounts({ config });
-	const [primaryAccount] = accounts ?? [];
+	const primaryAccount = useDefaultAccount();
+
 	const { balance, isLoading } = useBalance({
-		address: primaryAccount.address,
+		address: primaryAccount?.address,
 		config,
 		query: {
-			enabled: Boolean(primaryAccount.address) && !hideBalance,
+			enabled: Boolean(primaryAccount?.address) && !hideBalance,
 		},
 	});
 
 	if (children) {
 		return children({
 			balance,
-			address: primaryAccount.address,
+			address: primaryAccount?.address || "",
```

Comment:
We should probably change api here in this case so it accepts undefined or address